Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/Base32Encoder.kt

```diff
@@ -0,0 +1,35 @@
+package de.rki.coronawarnapp.eventregistration
+
+import com.google.common.io.BaseEncoding
+
+/**
+ * Base32 Encoding wrapper around [BaseEncoding.base32]
+ */
+class Base32Encoder {
```

Comment:
What do you think about making this an extension function and using the same pattern als `decodeBase64()` / `base64()`, see `ByteString`.